Add option to provide custom ssl certificates during development (#5845)
facebook/create-react-app: docusaurus/docs/using-https-in-development.md

```

Note that the server will use a self-signed certificate, so your web browser will almost definitely display a warning upon accessing the page.

> You can also specify a custom certificate (available in `react-scripts@2.2.0` and higher).

Author: Need to bump the version - does `3.1.0` work?
```suggestion
You can also specify a custom certificate (available in `react-scripts@3.1.0` and higher).
```

Reviewer: Yes, @iansu how would you want to handle this? I think 3.1.0 sounds good to me.

Author: @mrmckeb @iansu Should we go with `3.1.0` and get this merged, or is there other stuff we need to wait for?

> If this is an issue you can also specify a custom certificate from `react-scripts@2.2.0` and higher.

Reviewer: ```suggestion
You can also specify a custom certificate (available in `react-scripts@2.2.0` and higher).
```

> To do this set `SSL_CRT_FILE` and `SSL_KEY_FILE` to the path of the certificate and key files in the same way you do for `HTTPS` above. You will also need to set `HTTPS=true`.

Reviewer: ```suggestion
To set a custom certificate, set the `SSL_CRT_FILE` and `SSL_KEY_FILE` environment variables to the path of the certificate and key files in the same way you do for `HTTPS` above. Note that you will also need to set `HTTPS=true`.
```

> e.g. `SSL_CRT_FILE=cert.crt` and `SSL_KEY_FILE=cert.key`

Reviewer: Can we add a code block example here, as above? It could be just for Linux/MacOS as long as you specify that.
